components/Calendar/ReactCalendar.tsx

```diff
@@ -0,0 +1,186 @@
+                            {format(date.justDate, 'MMMM dd, EEEE')}
+                        </div>
+                        {times.map(({ start, end }) => (
+                            <button
+                                className="text-black p-2 rounded-4xl border border-orange max-w-full w-64 h-8 items-center flex justify-center text-lg"
+                                key={start.toString()}
+                                type="button"
+                                onClick={() => handleButtonClick(start)}
+                            >
+                                {format(start, 'kk:mm')} - {format(end, 'kk:mm')}
+                            </button>
+                        ))}
+                    </div>
+                )}
+                {showForm && (
+                    <div className="flex flex-col w-full md:w-80 items-center h-calendarHeight rounded-3xl shadow-xl">
+                        <div className="bg-orange h-16 w-full rounded-t-3xl flex items-center justify-center text-black">
+                            {format(date.justDate, 'MMMM dd, EEEE')}
+                        </div>
+                        <button className="text-black flex w-10/12 mt-4" onClick={handleBackButtonClick}>
+                            <svg
+                                xmlns="http://www.w3.org/2000/svg"
+                                fill="none"
+                                viewBox="0 0 24 24"
+                                strokeWidth={2}
+                                stroke="currentColor"
+                                className="w-6 h-6 mr-1"
+                            >
+                                <path strokeLinecap="round" strokeLinejoin="round" d="M15 19l-7-7 7-7" />
+                            </svg>
+                            Назад кон време
+                        </button>
+                        <form onSubmit={handleFormSubmit} className="flex flex-col items-center gap-2 mt-4 w-10/12">
+                            <label htmlFor="race" className="w-full justify-start flex text-black">Раса на куче</label>
+                            <input
+                                id="race"
+                                type="text"
+                                placeholder="Доберман"
+                                className="text-base py-1 pl-4 border border-gray rounded-4xl text-black w-full mt-1 mb-3"
+                                required
+                            />
+
+                            <label htmlFor="select1" className="w-full justify-start flex text-black">Тип на прошетка</label>
+                            <select
+                                id="select1"
+                                className="py-1 pl-4 border border-gray rounded-4xl text-black w-full mt-1 mb-3"
+                                required
+                            >
+                                <option value="" disabled selected hidden>Социјализација</option>
```

Comment:
Fix console errors
![image](https://github.com/user-attachments/assets/4fa2065d-c8d5-4ce6-bea5-51057e30d829)

Reply:
Option value should not be empty.